app/src/main/jni/transition/PeerManager.c

```diff
@@ -277,7 +278,7 @@ Java_com_breadwallet_wallet_BRPeerManager_create(JNIEnv *env, jobject thiz,
                             earliestKeyTime);
         _peerManager = BRPeerManagerNew(&BR_CHAIN_PARAMS, _wallet, (uint32_t) earliestKeyTime, _blocks,
                                         (size_t) blocksCount,
-                                        _peers, (size_t) peersCount, (double) fpRate);
+                                        _peers, (size_t) peersCount); //, t(double) fpRate
```

Comment:
Hey @vsima  the new variable **(fpRate)** is what I was trying to add.  It looks like you commented it out.

Remember the goal is to inject fpRate (false positive rate) from the user preferences.  So, when they select it the **sync becomes much shorter**

Reply:
Pulling the correct branch fixed the build issue. we can revert this commit.